qdrant-landing/content/documentation/embeddings/nomic.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Nomic"
+weight: 1100
+---
+
+# Nomic
+
+The `nomic-embed-text-v1` model is an [open source](https://github.com/nomic-ai/contrastors) 8192 context length text encoder.
```

Comment:
While we like open source, the link is to the Nomic contrastor. If I understand correctly:

```suggestion
The `nomic-embed-text-v1` model is an open source [8192 context length]((https://github.com/nomic-ai/contrastors)) text encoder.
```